docs/UsingConfigManager/UsingConfigManager.xml

```diff
@@ -706,6 +706,70 @@ sudo -u hpcc cp /etc/HPCCSystems/source/NewEnvironment.xml /etc/HPCCSystems/envi
       environment XML file.</para>
 
       <sect2 role="brk">
+        <title>Backupnode</title>
+
+        <para>Backupnode allows you to back up Thor clusters at regular
+        intervals. The Backupnode component is a way to allow administrators
+        to manage the backupnode process without using a cron job.</para>
+
+        <para>To enable this functionality, you add the backupnode component,
```

Comment:
To enable this functionality,
would be better to say:
To configure scheduled Thor node backups,